arviz/plots/autocorrplot.py

```diff
@@ -1,14 +1,14 @@
 """Autocorrelation plot of data."""
 import numpy as np
 
-from .backends import check_bokeh_version
 from ..data import convert_to_dataset
 from .plot_utils import (
     _scale_fig_size,
     default_grid,
     xarray_var_iter,
     _create_axes_grid,
     filter_plotters_list,
+    get_plotting_method,
```

Comment:
method or function?

Reply:
I guess it is a function. Happy to rename to whatever is best